### PR TITLE
New expanded player design and bug fixes

### PR DESCRIPTION
**Pull request by ChenJesse**
New player design

Also fixes some bugs, where we put too much logic in `cellForRowAtIndexPath`, which is only called upon visible cells. In this case, it is logic for preparing the post.player, so I made a new method and call it whenever new posts are fetched from the API

### CODE REVIEW

**Comment by loganallen** [comment]
Could we make the prev and next button tap-ranges bigger?

**Comment by ChenJesse** [comment]
Done

File: Tempo/Views/ExpandedPlayerView.swift [hunk 1]
```diff
@@ -176,6 +174,16 @@ class ExpandedPlayerView: UIView, UIGestureRecognizerDelegate {
 				post.toggleLike()
 				delegate.didToggleLike!()
 			}
+		} else if hitView == nextButton {
+			if let _ = delegate.playNextSong {
```
Comment (dennisfedorko) [inline Tempo/Views/ExpandedPlayerView.swift]: delegate?.playNextSong?()

File: Tempo/Views/ExpandedPlayerView.swift [hunk 2]
```diff
@@ -176,6 +174,16 @@ class ExpandedPlayerView: UIView, UIGestureRecognizerDelegate {
...
+				delegate.playNextSong!()
+			}
+		} else if hitView == prevButton {
+			if let _ = delegate.playPrevSong {
+				delegate.playPrevSong!()
```
Comment (dennisfedorko) [inline Tempo/Views/ExpandedPlayerView.swift]: delegate?().playPrevSong?()

**Comment by dennisfedorko** [comment]
Looks good, just fix up those few comments and merge

**Pull request merged**